gradle/dependencies-graph.gradle

```diff
@@ -0,0 +1,25 @@
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath pluginDependencies.dependencyGraph
+    }
+}
+
+import com.vanniktech.dependency.graph.generator.DependencyGraphGeneratorPlugin
+import com.vanniktech.dependency.graph.generator.DependencyGraphGeneratorExtension.Generator
+
+
+plugins.apply(DependencyGraphGeneratorPlugin)
+
+def mapboxGenerator = new Generator(
+        "mapboxLibraries", // Suffix for our Gradle task.
+        { dependency -> dependency.getModuleGroup().startsWith("com.mapbox") }, // Only want Mapbox libs.
+        { dependency -> true }, // Include transitive dependencies.
```

Comment:
Technically that's the default and does not need to be passed. I've done so in the example since I need the next parameter to configure and in Groovy I don't think you can use named parameters with default values.